elliottlib/cli/get_golang_versions_cli.py

```diff
@@ -72,19 +72,19 @@ def get_nvrs_golang(nvrs, logger):
         else:
             rpm_nvrs.append(nvr_tuple)
 
-    nvrs = {}
     if rpm_nvrs:
-        nvrs.update(util.get_golang_rpm_nvrs(rpm_nvrs, logger))
```

Comment:
This was simpler when we were returning nvrs with their go version, now that the method
signatures have changed to returning go_nvr_map - to support this branch - printing image and rpm nvrs 
has become more tedious - instead just print them separate.